ddtrace/tracer.py

```diff
@@ -717,3 +717,7 @@ def _is_agentless_environment():
             # We are in an AWS Lambda environment
             return True
         return False
+
+    @staticmethod
+    def _is_span_internal(span):
+        return not span.span_type or span.span_type in _INTERNAL_APPLICATION_SPAN_TYPES
```

Comment:
if you expect `None` to be seen as internal I'd rather write this explicitely `span.span_type is None`